### PR TITLE
Rename order field in LevelType (2/3)

### PR DESCRIPTION
**Pull request by pwnage101**
Stage 2: Start using the new field, remove the old field, and copy the
values of the old column into the new column.  However, this release
DOES NOT include the migration to actually delete the old column.

DE-1829

### CODE REVIEW

File: course_discovery/apps/course_metadata/migrations/0218_leveltype_sort_value_copy_values.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_metadata', '0217_leveltype_sort_value'),
+    ]
+
+    operations = [
+        migrations.RunPython(
+            copy_column_values_forwards,
+            copy_column_values_backwards,
```
Comment (mikix) [inline course_discovery/apps/course_metadata/migrations/0218_leveltype_sort_value_copy_values.py]: You can just specify `migrations.RunPython.noop` for brevity on the backwards roll.

**Review by mikix — APPROVED** [review]

File: course_discovery/apps/course_metadata/migrations/0218_leveltype_sort_value_copy_values.py (hunk 1 above)
Comment (pwnage101) [reply course_discovery/apps/course_metadata/migrations/0218_leveltype_sort_value_copy_values.py]: oh, nice, will do.

**Review by macdiesel — APPROVED** [review]

**Comment by pwnage101** [comment]
Tests passed locally, so i'm closing/reopening this ticket to force travis to restart.

**Comment by Dillon-Dumesnil** [comment]
> Tests passed locally, so i'm closing/reopening this ticket to force travis to restart.

Yeah, we have problems with some flakey tests that we are hoping to take care of soon. If you find it easier, you can also hop into travis and rerun the build or job there so you don't have to close and reopen the PR.

**Comment by pwnage101** [comment]
> If you find it easier, you can also hop into travis and rerun the build or job there

Yea, I couldn't find the button, so they either moved it or I don't have permissions.

**Comment by hunytalk** [comment]
> > If you find it easier, you can also hop into travis and rerun the build or job there
> 
> Yea, I couldn't find the button, so they either moved it or I don't have permissions.

I think you weren't logged in to travis that's why you couldn't see cancel / rerun option. I could see cancel build option at the moment

**Review by hunytalk — APPROVED** [review]

**Pull request merged**